lib/elixir/lib/integer.ex

```diff
@@ -189,13 +189,14 @@ defmodule Integer do
   @spec digits(integer, pos_integer) :: [integer, ...]
   def digits(integer, base \\ 10)
       when is_integer(integer) and is_integer(base) and base >= 2 do
-    do_digits(integer, base, [])
+    digits(integer, base, [])
   end
 
-  defp do_digits(integer, base, acc) when abs(integer) < base, do: [integer | acc]
+  defp digits(0, _base, []), do: [0]
```

Comment:
It might not have much impact on the bench, but I think we can remove this check out from the recursive function and check it once in the main function?

Reply:
So you think to write it as in digits/2?
```elixir
if integer == 0, do: [0], else: digits(integer, base, [])
```

Reply:
Exactly 👍 (I would use `case` though)

Reply:
Makes sense. I've changed it now. Thanks for sharing your opinion (:

Reply:
Perfect 💯